www/src/utils/ogFont.ts

```diff
@@ -0,0 +1,41 @@
+export async function getFont<TWeights extends readonly number[]>({
```

Comment:
looks like you snagged this from my site, please ref it back :)

```suggestion
/** https://github.com/juliusmarminge/jumr.dev/blob/main/app/og-image/get-fonts.ts */
export async function getFont<TWeights extends readonly number[]>({
```

(also you can use const params now in TS5)

Reply:
I actually found it on the nextra branch that you send in Discord. I will add a link to your site though.